docs/config.md

```diff
@@ -1,100 +1,272 @@
-# Config system
+- [Config](#config)
+  - [Basic](#basic)
+  - [Interpolation](#interpolation)
+    - [Callable Interpolation](#callable-interpolation)
+    - [Interpolation by Path](#interpolation-by-path)
+    - [Interpolation by Attribute](#interpolation-by-attribute)
+    - [Default Value by Path Interpolation](#default-value-by-path-interpolation)
+    - [Ignore Interpolation](#ignore-interpolation)
+    - [Lazy Evaluation](#lazy-evaluation)
 
-Algorithm configurations such as training hyperparameters are serialized as nested mappings often using the YAML format.  
-Nested mappings are parsed by `pytorch_pfn_extras.config.Config` that returns a dictionary of objects that are lazily evaluated.
-The most notable feature is a special handling of the `type` key, which invokes a function or instantiates an object. The arguments for them are mapped by non-`type` keys.
-```python 
-import yaml
+
+# Config
+
+## Basic
+Config object is created from a dictionary.
```

Comment:
Config also supports a list, a string, and a number.

Reply:
I deleted this line